### ISSUE TITLE
Internal error during ready callback: 'MCU_TMC_uart' object has no attribute 'get_tmc_frequency

### ISSUE THREAD

**Issue by LilDominus** [andrewmcgr/klipper_tmc_autotune#64]
Hello, I'm trying to configure auto_autotune on my Ender 6 using SKR mini E3 V3 running v0.0.0-4766-gdgus-reloaded-230526-0-gbaae7f3a-inferred. I encounter this error. One guy on discord said that files need from tmc_autotune is  not yet merge to gdgus-reloaded. Is there anything I can do to resolve this issue? Thank a lot and have a good day!
![Untitled](https://github.com/andrewmcgr/klipper_tmc_autotune/assets/86697225/3ab79711-73bd-481b-869d-14660d23108e)

**Comment by Drachenkaetzchen** [comment]
The error comes from an outdated Klipper version. Update Klipper and try again

**Comment by Frix-x** [comment]
Right, the gdgus fork of Klipper that you are using is currently 355 commits behind Klipper main branch and date from more than a year ago. If you want to continue using this fork, you will need to ask his maintainer to update it.

This plugin is designed around the official Klipper repository

**Comment by mjonuschat** [comment]
Closing this as it's due to an outdated and unsupported klipper fork.
